doc/geany.txt

```diff
@@ -3719,9 +3719,15 @@ Clone                                                     See `Cloning documents
 
 Replace tabs with space                                   Replaces all tabs with the right amount of spaces
                                                           in the whole document, or the current selection.
+                                                          If the replacement was made in the whole document,
+                                                          also changes the document's indent type to
+                                                          *Spaces*.
```

Comment:
Yes. But the added sentence refers to the previous sentence:

"Replaces all tabs with the right amount of spaces in the whole document, or the current selection."

So by "whole document" it means "no selection is made". Just let me know if you think some different wording should be used.